packages/expo-modules-autolinking/android/expo-gradle-plugin/README.md

```diff
@@ -0,0 +1,26 @@
+# expo-modules-plugin
+
+This project contains two Gradle plugins that are used to inject the necessary dependencies and configurations into an Android project that uses Expo modules. It also provides a shared project that contains common code for both plugins.
+
+### `expo-autolining-settings-plugin`
+
+The settings plugin is an entry point for our setup. It should be applied to the root `settings.gradle` file of the application. 
+
+Responsibilities:
+- add all modules into the project hierarchy; modules won't be added to the dependency graph. We decided that the `expo` package will depend on them rather than adding them directly to the app project
+- add extra Maven repositories
+- link and apply custom plugins
+- expose autolinking configuration
```

Comment:
```suggestion
Responsibilities:
- Add all modules into the project hierarchy; modules won't be added to the dependency graph. The `expo` package will depend on them rather than adding them directly to the app project.
- Add extra Maven repositories.
- Link and apply custom plugins.
- Expose autolinking configuration.
```